HACKING.md

```diff
@@ -0,0 +1,46 @@
+# Cluster DNS Operator Hacking
+
+
+## Local development
+
+It's possible (and useful) to develop the operator locally targeting a remote cluster.
+
+### Prerequisites
+
+* An OpenShift cluster with at least a master, and compute node. 
+* An admin-scoped `KUBECONFIG` for the cluster.
+* The [operator-sdk](https://github.com/operator-framework/operator-sdk).
+
+#### GCP test clusters
+
+One reliable and hands-free way to create a suitable test cluster and `KUBECONFIG` in GCP is to use the [openshift/release](https://github.com/openshift/release/tree/master/cluster/test-deploy) tooling.
+
+### Building
+
+To build the operator during development, use the standard Go toolchain:
+
+```
+$ go build ./...
+```
+
+### Running
+
+To run the operator, first deploy the custom resource definitions:
+
+```
+$ oc create -f deploy/crd.yaml
+```
+
+Then, use the operator-sdk to launch the operator:
+
+```
+$ operator-sdk up local namespace default --kubeconfig=$KUBECONFIG
+```
+
```

Comment:
maybe add instructions for containerized operator
operator-sdk build openshift/cluster-dns-operator
oc create -f operator.yaml